src/org/zaproxy/zap/extension/bugtracker/BugTrackerBugzilla.java

```diff
@@ -0,0 +1,392 @@
+            }
+            if(alert.getDescription().length() > 0 ) {
+                description.append(" *DESCRIPTION* \n" + alert.getDescription().toString() + "\n\n");
+            }
+            if(alert.getOtherInfo().length() > 0 ) {
+                description.append(" *OTHER INFO* \n" + alert.getOtherInfo().toString() + "\n\n");
+            }
+            if(alert.getSolution().length() > 0 ) {
+                description.append(" *SOLUTION* \n" + alert.getSolution().toString() + "\n\n");
+            }
+            if(alert.getReference().length() > 0 ) {
+                description.append(" *REFERENCE* \n" + alert.getReference().toString() + "\n\n");
+            }
+            if(alert.getParam().length() > 0 ) {
+                description.append(" *PARAMETER* \n" + alert.getParam().toString() + "\n\n");
+            }
+            if(alert.getAttack().length() > 0 ) {
+                description.append(" *ATTACK* \n" + alert.getAttack().toString() + "\n\n");
+            }
+            if(alert.getEvidence().length() > 0 ) {
+                description.append(" *EVIDENCE* \n" + alert.getEvidence().toString() + "\n\n");
+            }
+            if(alert.getRisk() >= 0 ) {
+                description.append("Risk: " + alert.MSG_RISK[alert.getRisk()] + ", ");
+            }
+            if(alert.getConfidence() >= 0 ) {
+                description.append("Conf: " + alert.MSG_CONFIDENCE[alert.getConfidence()] + ", ");
+            }
+            if(alert.getCweId() >= 0 ) {
+                description.append("CWE: " + alert.getCweId() + ", ");
+            }
+            if(alert.getWascId() >= 0 ) {
+                description.append("WASC: " + alert.getWascId() + "\n\n\n\n ");
+            }
+
+        }
+        descriptionIssue = description.toString();
+	}
+
+	public String getSummary() {
+		return this.summaryIssue;
+	}
+	
+	public String getDesc() {
+		return this.descriptionIssue;
+	}
+
+
+	public String raiseOnTracker(String url, String summary, String description, String product, String component, String version, String os, String platform, String username, String password) throws IOException {
+		try {
+			BugzillaConnector conn = new BugzillaConnector(); 
+			conn.connectTo(url); 
+
+			LogIn logIn = new LogIn(username, password); 
+			conn.executeMethod(logIn); 
+
+			Bug bug = new BugFactory().newBug() 
+				    .setProduct(product) 
+				    .setComponent(component) 
+				    .setVersion(version) 
+				    .setPlatform(platform) 
+				    .setOperatingSystem(os) 
+				    .setDescription(description) 
+				    .setSummary(summary) 
+				    .createBug(); 
+
+			ReportBug report = new ReportBug(bug); 
+			conn.executeMethod(report); 
+            return null;
+		} catch(ConnectionException e) {
+            log.debug(e.toString());
+            String[] response = e.toString().split("com.j2bugzilla.base.ConnectionException: ");
+            return response[1];
+		} catch(BugzillaException e) {
```

Comment:
Does this help? I'd have thought just passing on the exceptions would be easier to manage...